config/locales/en.yml

```diff
@@ -1801,10 +1801,17 @@ user_mailer.account_reset_request.cancel: Don’t want to delete your account? S
 user_mailer.account_reset_request.header: Your account will be deleted in %{interval}
 user_mailer.account_reset_request.intro_html: 'As a security measure, %{app_name} requires a two-step process to delete your account:<br><br> Step One: There is a waiting period of %{waiting_period} if you have lost access to your authentication methods and need to delete your account. If you locate your authentication methods, you can sign in to your %{app_name} account to cancel this request.<br><br> Step Two: After the waiting period of %{waiting_period}, you will receive an email that will ask you to confirm the deletion of your %{app_name} account. Your account will not be deleted until you confirm.'
 user_mailer.account_reset_request.subject: How to delete your %{app_name} account
+user_mailer.account_verified_but_not_connected.change_password_link: change your password
+user_mailer.account_verified_but_not_connected.contact_link: contact us
+user_mailer.account_verified_but_not_connected.help_html: If you did not perform this action, please %{contact_link_html} and sign in to %{change_password_link_html}.
+user_mailer.account_verified_but_not_connected.instructions_html: Sign back in at the <strong>%{sp_name}</strong> website to connect your verified information and access services.
+user_mailer.account_verified_but_not_connected.intro: You verified your identity with %{app_name} on %{date}
+user_mailer.account_verified_but_not_connected.subject: '[Action required] Connect your information to %{sp_name}.'
 user_mailer.account_verified.change_password_link: change your password
 user_mailer.account_verified.contact_link: contact us
-user_mailer.account_verified.intro_html: You successfully verified your identity with %{sp_name} on %{date} using %{app_name}. If you did not perform this action, please %{contact_link_html} and sign in to %{change_password_link_html}.
-user_mailer.account_verified.subject: You verified your identity with %{sp_name}.
+user_mailer.account_verified.help_html: If you did not perform this action, please %{contact_link_html} and sign in to %{change_password_link_html}.
+user_mailer.account_verified.intro_html: On %{date}, you used %{app_name} to:<ul><li>Verify your identity</li><li>Connect your verified information to %{sp_name}</li></ul>
```

Comment:
in the spirit of moving HTML into ERBs, I think it would be worth considering moving each sentence/bullet into its own key, and then leave the HTML markup in the template